openadapt/db/crud.py

```diff
@@ -530,7 +544,13 @@ def get_screenshots(
     Returns:
         list[Screenshot]: A list of screenshots for the recording.
     """
-    screenshots = _get(session, Screenshot, recording.id)
+    screenshots = _get(
+        session,
+        Screenshot,
+        recording.id,
+        eager=True,
+        relationships=["action_event", "recording"],
```

Comment:
Why not `relationships=["action_event", "recording", "window"]`?

Reply:
Screenshot has no attribute `window`